### ISSUE TITLE
feature:  wiki text sections embedded on website

### ISSUE THREAD

**Issue by carchrae** [foodcoops/foodsoft#652]
in our co-op, communication can be a challenge, especially when there are many people sharing jobs.  despite writing manuals, training, and so on, people will often not read, update or just forget important things.

one solution i thought about for this is to allow our co-op to embed a wiki page on each website page.  this means that each co-op member can add specific instructions to the page, for example, perhaps some part of the process has changed, some supplier has a new rule, or so on.

here is an example of what i mean - i currently have only implemented this on the home page, but i would like to add it to other pages instead of adding custom text in templates, and so on.

in terms of developing this for a PR, i propose to suggest a pattern to determine the wiki page based on the url/route.  then, on each page, there would be a blank page and a small 
![image](https://user-images.githubusercontent.com/959825/55640807-38643600-5781-11e9-9f95-d7147c57ed3d.png) icon.  any co-op member can then edit the page.  

personally, i am not in favour of adding rules/roles for who can edit the text, for the same reasons that anyone can edit the wiki.  there is already a lot of trust in the co-op, and if someone writes text that is malicious, we know who did it, and it is also easy to revert (because it uses the wiki)

![image](https://user-images.githubusercontent.com/959825/55640452-77de5280-5780-11e9-870e-bf379a8e2d50.png)

**Comment by carchrae** [comment]
this is the wiki page that is embedded above

![image](https://user-images.githubusercontent.com/959825/55640961-8f6a0b00-5781-11e9-9366-34c1d3ba3a73.png)

**Comment by wvengen** [comment]
That sounds like an interesting, and very useful idea.
You show an example of a side-menu. I think an optional top-banner/message would also be useful.

As a follow-up feature: for orders, one might even want to customize the text based on supplier.

**Comment by carchrae** [comment]
yes indeed!  the general idea was you could have one on most pages.  with a very unobtrusive little edit button on the top.

i like the idea of the specific ones per supplier too.  (somewhat related, i've added a 'supplier_note' field in my branch - because i wanted custom instructions emailed to each supplier, but this does not use the wiki https://github.com/carchrae/foodsoft/commit/dea1b2ef1774cdd293f3d4d90c24efb68db72aa1  - i will get back into submitting patches for these, but i had to get a lot of changes working for our co-op and do that quickly)
